README.rst

```diff
@@ -340,6 +340,60 @@ unconventional syntax (without the usual ``@``):
     if isinstance(res, Ok):
         print(res.value)
 
+
+Do notation: syntactic sugar for a sequence of `and_then()` calls.
+Much like the equivalent in Rust or Haskell, but with different syntax.
+Instead of `x <- Ok(1)` we write `for x in Ok(1)`.
```

Comment:
You used markdown syntax here, the backticks are different in rst. There's other examples in this file for how to use backticks in rst

Reply:
Also look at the rendered preview in the github diff to double check the syntax. It's very helpful.

Reply:
![image](https://github.com/rustedpy/result/assets/15352422/d29225f2-4f6d-4378-bd6f-de55867df70b)